en/what-do-i-need-to-publish.rst

```diff
@@ -0,0 +1,31 @@
+What do I need to publish?
+=========
+
+Publishing to IATI means you are telling a story about the work your organisation does. For this you break your work down into activities, these could be your programmes or projects. For each activity you detail what the work is, where and when it is happening, who is involved and how it is being financed etc. Further detail can then be added e.g. geographic co-ordinates, results information, document links and policy markers. The IATI technical team recommends that you start by adding the details (in the form of elements) below:
+
+| Topic  | Element | Mandatory | Recommended |
+| --- | --- | :---: | :---: |
+| Identification  | IATI Identifier  | Yes |
+|  | Reporting Org | Yes |
+|Basic Activity Information | Title | Yes |
+|  | Description | Yes |
+|  | Activity Status | Yes |
+|  | Activity Date | Yes |
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+| Classifications | Sector | Yes |
+| Financial | Budget |  | Yes |
+|  | Transactions |  | Yes |
+
+A full table of elements that can be reported in IATI is available here:
+
+* `All elements in IATI <http://iatistandard.org/202/guidance/what-do-i-need-to-publish/list-of-all-elements>`__
```

Comment:
Does such a table exist on the standard website?

There’s http://iatistandard.org/202/activity-standard/elements/ … But a) that’s the activity standard, and b) I’m not sure whether it’s okay to link to version-specific stuff in the guidance?